src/aspire/source/coordinates.py

```diff
@@ -325,6 +325,14 @@ def _images(self, start=0, num=np.inf, indices=None):
 
         return Image(im)
 
+    @staticmethod
+    def _is_number(text):
```

Comment:
I thought surely something better was built in, but googling around I see this is probably one of the more performant ways to check this. Interesting.

Reply:
> Cool, thanks. So the issue is that some of our input data has floats for (box) sizes? That's interesting, I thought they were pixels....

I've only seen this for coordinates but I decided to just allow the box size to be a float for thoroughness. (I can image a situation where someone puts `256.0000` as the size)

> Just a remark. Generally when testing cases like this it is better to split the test into different cases. At least when it is simple to do so. In this case it might be better to have one case for integers, one for floats, and one for junk. Right now ints/floats are mixed together so you might not immediately know what breaks the UT. I'm not sure this case merits refactoring, just something to consider next time.

No you're absolutely right I'll change it into a single small test rather than mixing it like this. Thanks